x/slashing/client/cli/query.go

```diff
@@ -51,16 +52,16 @@ $ <appd> query slashing signing-info cosmosvalconspub1zcjduepqfhvwcmt7p06fvdgexx
 				return err
 			}
 
-			queryClient := types.NewQueryClient(clientCtx)
-
-			pk, err := sdk.GetPubKeyFromBech32(sdk.Bech32PubKeyTypeConsPub, args[0])
-			if err != nil {
+			var pk cryptotypes.PubKey
+			am := codec.NewJSONAnyMarshaler(clientCtx.JSONMarshaler, clientCtx.InterfaceRegistry)
+			if err := codec.UnmarshalIfcJSON(am, &pk, []byte(args[0])); err != nil {
 				return err
 			}
 
 			consAddr := sdk.ConsAddress(pk.Address())
 			params := &types.QuerySigningInfoRequest{ConsAddress: consAddr.String()}
-			res, err := queryClient.SigningInfo(context.Background(), params)
+			queryClient := types.NewQueryClient(clientCtx)
+			res, err := queryClient.SigningInfo(cmd.Context(), params)
```

Comment:
the example in this cmd needs to be changed too.

Reply:
done:
{"@type":"/cosmos.crypto.ed25519.PubKey","key":"OauFcTKbN5Lx3fJL689cikXBqe+hcp6Y+x0rYUdR9Jk="}